docs/CLI.md

```diff
@@ -1,235 +1,3 @@
-Bcoin ships with bcoin-cli as its default HTTP client for command line access.
+Bcoin ships with [bclient](https://github.com/bcoin-org/bclient) as its default HTTP client for command line access.
 
-## Configuration
```

Comment:
Some of this configuration information isn't available using the help provided by `./bin/bcoin-cli`, and could be worth keeping here?

Reply:
For this page, I started updating then realized all this info is already online at `api-docs`, with much better details, but maybe I could leave a few quick examples, and recommend the `help` command for more.